Stripe/STPPaymentMethodsInternalViewController.m

```diff
@@ -53,20 +52,14 @@ - (instancetype)initWithConfiguration:(STPPaymentConfiguration *)configuration
     return self;
 }
 
-- (void)viewDidLoad {
-    [super viewDidLoad];
-    self.automaticallyAdjustsScrollViewInsets = NO;
-    
-    UITableView *tableView = [[UITableView alloc] initWithFrame:self.view.bounds style:UITableViewStyleGrouped];
-    tableView.allowsMultipleSelectionDuringEditing = NO;
-    tableView.dataSource = self;
-    tableView.delegate = self;
-    [tableView registerClass:[STPPaymentMethodTableViewCell class] forCellReuseIdentifier:STPPaymentMethodCellReuseIdentifier];
-    tableView.sectionHeaderHeight = 30;
-    tableView.separatorStyle = UITableViewCellSeparatorStyleNone; // use fake separator views
-    tableView.separatorInset = UIEdgeInsetsMake(0, 18, 0, 0);
-    self.tableView = tableView;
-    [self.view addSubview:tableView];
+- (void)createAndSetupViews {
+    [super createAndSetupViews];
+
+    self.tableView.allowsMultipleSelectionDuringEditing = NO;
```

Comment:
Nowhere else makes these changes so I wasn't sure? Also `allowsMultipleSelectionDuringEditing` is NO by default anyway.